patches/api/0304-Missing-Entity-Behavior-API.patch

```diff
@@ -1176,16 +1177,16 @@ index 7cc1d9a966454af70b7c25735fe474fe3eb97eb4..ad2eaee347cd2864aef30d2af48c1be6
 +    // Paper stop - missing entity behaviour api - converting without entity event
  }
 diff --git a/src/main/java/org/bukkit/entity/minecart/ExplosiveMinecart.java b/src/main/java/org/bukkit/entity/minecart/ExplosiveMinecart.java
-index a4411daca0d8a770b0d15e08847b82df8a3ec27f..50464aa8c38cd47fc049ee022e25cb4976260fa0 100644
+index a4411daca0d8a770b0d15e08847b82df8a3ec27f..114741151c12c0c195451e274309384911f92270 100644
 --- a/src/main/java/org/bukkit/entity/minecart/ExplosiveMinecart.java
 +++ b/src/main/java/org/bukkit/entity/minecart/ExplosiveMinecart.java
-@@ -6,4 +6,19 @@ import org.bukkit.entity.Minecart;
+@@ -6,4 +6,44 @@ import org.bukkit.entity.Minecart;
   * Represents a Minecart with TNT inside it that can explode when triggered.
   */
  public interface ExplosiveMinecart extends Minecart {
 +    // Paper start - Entity API
 +    /**
-+     * Set the number of ticks until the minecart explodes after being primed.
++     * Set the number of ticks until the minecart explodes after being primed
```

Comment:
Javadocs seem to differ from place to place, there's no standard that I'm aware of? Even in the present patch, the setter has a dot, and the getter doesn't, and since I usually don't place a dot if there's only one sentence, I removed it.

But as per request placed dots instead! :)